src/components/common/Header/@setting/layouts/setting.tsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import styled from "styled-components";
+import Image from "next/image";
+import { Back } from "@/assets/images";
+import { theme, flex, font } from "@/styles";
+
+interface SettingModalProps {
+  onClose: () => void;
+}
+
+const SettingModal = ({ onClose }: SettingModalProps) => {
+  return (
+    <Modal>
+      <ModalContent>
+        <SettingBox>
+          <SettingText>설정</SettingText>
+          <button onClick={onClose}>
+            <ExitButton src={Back} alt="Back" />
+          </button>

```

Comment:
불필요한 element가 많이 사용되고 있는 것 같습니다!
위에서 말씀드린 것처럼, 컴포넌트화되어있는 svg인 XIcon을 사용하면 이를 더 깔끔하게 해결할 수 있으며,
XIcon은 svg props를 그대로 받아 넣어주기에 바로 props를 넣어 사용할 수 있습니다!

```suggestion
<XIcon onClick={onClose} />
```